test/esinstall/import-nothing/import-nothing.test.js

```diff
@@ -0,0 +1,15 @@
+const {install} = require('../../../esinstall/lib');
+
+describe('error-no-dep-list', () => {
+  it('importing nothing', async () => {
+    expect(() =>
+      install(
+        [
+          /* nothing! */
+        ],
+        {cwd: __dirname},
+      ),
+    ).rejects.toThrowError(`No ESM dependencies found!
```

Comment:
Most errors throw "Install failed" but then logger.error the actual error message. CLI legacy... yayyyyyy.

It would be good to update to actually throw some better info than just "install failed" OR attach the logger.error'd message as `error.reason`.